db/projects/cindicator/project.yaml

```diff
@@ -0,0 +1,66 @@
+name: Cindicator
```

Comment:
Are you sure about their logos? Because when I check for example their twitter, the logo is totally different

https://pbs.twimg.com/profile_images/1052983470711066625/aXwEXsZz_400x400.jpg

Reply:
They probably chenged logo (it is like that even on website) - so I dig it up. But I still leave old logo as an alternative